media_site/media_site/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for media_site project.
+
+Generated by 'django-admin startproject' using Django 4.2.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+from dotenv import load_dotenv
+import os 
+
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.getenv("DEBUG", False)
+
+ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS", "*").split(",")
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'rest_framework',
+]
```

Comment:
We have to maintain following structure:

DJANGO_APPS, THIRD_PARTY_APPS, INHOUSE_APP

and we have to add these INSTALLED_APPS

example of,
DJANGO_APPS is 'django.contrib.admin', 'django.contrib.auth'  and so on.
THIRD_PARTY_APPS is 'rest_framework', 'channels' and so on.
INHOUSE_APP is 'accounts', 'projects' and so on. (which we are creating using django startapp command)